scripts/driver/driver_internal.sh

```diff
@@ -47,7 +47,6 @@ setup_kubeconfig.sh
 
 # Extract schema and values files.
 docker run \
```

Comment:
Fun fact, if you don't include -t, you cant ctrl+c this.

Reply:
Needs a ctrl-d, i believe.